### DISCUSSION TITLE
Extremely Need Chinese App Store Release

### DISCUSSION THREAD

**Discussion by OneCuriousLearner** [thebaselab/codeapp#998] [Q&A]
Just as what I write in the title.
As a college student who major in CS in China,
I’ve been looking for a suitable coding App in App Store for a really long time,
when I got words of the Code App, I rush to my App Store but find the App unsupported in Chinese App Store………
Tons of students and programmers who use iPad are thristy for such an App,
it can be previewed that the solution can bring a happy end to everyone!
Sincerely waiting for your reply…

**Comment by bummoblizard** [comment]
Do you mean Code App unavailable for purchase now on the Chinese App Store? I wasn't aware of it. Can you send me a screenshot of it?

**Reply by OneCuriousLearner** [reply]
When I search for the App several days later, it’s still can not be found in my App Store, but the time I received your reply, I’ve magically found it!(crying with smile)Thanks for your speedily reply,I’m downloading it! Haha!
By the way, could you please offer some Instructions for Use? The bottoms and Accelerator keys are still unknown for me…And I wish more and more Plugins could be available in this wonderful App.

**Reply by OneCuriousLearner** [reply]
I’m sorry for the wrong word “later” in the first sentence,it should be “before”…

**Reply by bummoblizard** [reply]
Great to hear! The arrow keys are for navigating the code. Here's some documentation of the app: https://code.thebaselab.com. You might also create issues or discussions here for help.

**Reply by OneCuriousLearner** [reply]
Thanks! I will try to learn!
